frontend/src/setupProxy.js

```diff
@@ -0,0 +1,11 @@
+const { createProxyMiddleware } = require("http-proxy-middleware");
+
+module.exports = function(app) {
+    app.use(
+        "/api",
+        createProxyMiddleware({
+            target: process.env.HTTP_PROXY,
+            changeOrigin: true
+        })
+    );
+};
```

Comment:
Why do you need this? Why not use `proxy` in `package.json`?

Reply:
I think I tried it but found no success using that method

Reply:
Check `mat`. This is unnecessary complexity atm

Reply:
Unsure why it didn't work last time it worked but `proxy` seems to function now.